homeassistant/components/aladdin_connect/cover.py

```diff
@@ -61,50 +61,22 @@ def setup_platform(
 class AladdinDevice(CoverEntity):
     """Representation of Aladdin Connect cover."""
 
+    _attr_device_class = "garage"
```

Comment:
```suggestion
    _attr_device_class = DEVICE_CLASS_GARAGE
```
`from homeassistant.components.cover import DEVICE_CLASS_GARAGE`